src/Features/Core/Portable/Experimentation/ReSharperStatus.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Experimentation
+{
+    internal enum ReSharperStatus
```

Comment:
Because I didn't know any better :).

---
In reply to: [155655884](https://github.com/dotnet/roslyn/pull/23605#discussion_r155655884) [](ancestors = 155655884)